lua/tests/automated/utils_spec.lua

```diff
@@ -297,4 +297,10 @@ describe("transform_path", function()
       new_relpath "init.lua lua/telescope"
     )
   end)
+
+  it("handles function passed to path_display", function()
+    assert_path(function(_, path)
```

Comment:
I don't think the `assert_path` function supports function type `path_display`s yet.
So this new test case won't actually do anything. If you can add support for it, I'd appreciate it.

Reply:
I added it, but forgot to commit...